source/_posts/2021-04-07-release-20214.markdown

```diff
@@ -2801,7 +2801,7 @@ The following integrations are no longer available as of this release:
 [todoist docs]: /integrations/todoist/
 [toon docs]: /integrations/toon/
 [tplink docs]: /integrations/tplink/
-[trace docs]: /integrations/trace/
+[trace docs]: /docs/automation/troubleshooting/
```

Comment:
This is generated and points to the trace integration. We don't have documentation in place for the trace integration currently, however, we can redirect the integration page to the docs.

Reply:
Done